package.json

```diff
@@ -12,8 +12,8 @@
   },
   "scripts": {
     "start": "element-server",
-    "lint": "eslint --quiet .",
-    "lint-strict": "eslint .",
+    "lint": "eslint --max-warnings=0 .",
+    "lint-fix": "eslint --fix .",
```

Comment:
FYI @klebba , I think these are the two linting scripts we'll want moving forward.